ssz/sedes/boolean.py

```diff
@@ -33,6 +33,9 @@ def deserialize(self, data: bytes) -> bool:
                 f"{encode_hex(data)})",
             )
 
+    def chunk_count(self) -> int:
+        return 1
```

Comment:
do we want to explicitly define this here? seems like we can rely on the parent implementation